### PR TITLE
Adding Grafana SMTP config to documentation

### PR DESCRIPTION
**Pull request by alimez**

### CODE REVIEW

**Comment by alimez** [comment]
fixes #835 835

File: docs/user/install.md [hunk 1]
```diff
@@ -114,3 +116,44 @@ using Ubuntu 18.04, though other versions of Linux should be compatible.
 *   Otherwise, if you are running from a different machine you'll need to copy
     the Turbinia config from the original machine, or from the Turbinia server
     from `/etc/turbinia/turbinia.conf`.
+
+### Grafana SMTP Setup 
+
+If you want to receive alert notifications from Grafan, you'll need to setup a SMTP server for Grafana. To configure a SMTP server, you need to add the following environment variables to `Grafana` `env` section in `osdfir-infrastructure/modules/monitoring/main.tf`
```
Comment (hacktobeer) [inline docs/user/install.md:122]: GrafanA

**Review by hacktobeer — APPROVED** [review]
smallest nit ever ;)

File: docs/user/install.md (hunk 1 above)
Comment (alimez) [reply docs/user/install.md]: done

**Review by aarontp — APPROVED** [review]
LGTM

**Pull request merged**